taskcluster/test/params/train.yml

```diff
@@ -0,0 +1,94 @@
+base_ref: origin/main
```

Comment:
I wonder if we actually need to provide parameters for this use case? AFAICT, these are exactly the same as the defaults in `parameters.py`, so anytime those change this will become a less good test.

Reply:
Yeah it is the defaults. Maybe there's a better way to do a test like this?

It's rather rudimentary but it has proven useful in the VPN project (and Firefox.) 

https://searchfox.org/mozilla-central/source/taskcluster/ci/source-test/taskgraph.yml#32

I hear you tho, they are brittle and do break :/

https://phabricator.services.mozilla.com/D187137

Reply:
I think providing separate parameters is useful mostly in cases when you have multiple distinct sets of tasks you run (eg: you do different things for pull requests, pushes, releases, etc.). In this repository, we expect the same tasks each time - nothing in the parameters changes which tasks are being targeted, it just changes some payloads.

I'm not necessarily opposed to keeping this - I'm just struggling to find a clear benefit vs. just running taskgraph without a parameters file specified.

Reply:
I see. So `taskgraph full` without parameters is more or less the same thing as running the action with the defaults? We are adding things on pushes and pull requests (@gregtatum is adding linting and he moved the snakemake dry run into the Taskgraph pipeline)

Reply:
Unless I'm mistaken, it should be exactly the same, because [the defaults for the actions](https://github.com/mozilla/firefox-translations-training/blob/cce2cd7b7c98bee6512a8ccfd24b98e1ac51ff6d/taskcluster/translations_taskgraph/actions/train.py#L9) are sourced from [the defaults for the parameters.](https://github.com/mozilla/firefox-translations-training/blob/cce2cd7b7c98bee6512a8ccfd24b98e1ac51ff6d/taskcluster/translations_taskgraph/parameters.py#L13) (Things like build date and revision would be different - but we don't use those when generating the graph.)

Reply:
Oh I see. Now that you mention it I see the default action configuration in all the parameter files 0: